jenkins/pipeline.groovy

```diff
@@ -85,6 +85,21 @@ pipeline {
                 }
 
                 sh 'echo Prepare: \$(date -u "+%s")'
+                echo 'Checking Percona Server branch version, JEN-913 prevent wrong version run'
+                sh '''
+                    MY_BRANCH_BASE_MAJOR=5
+                    MY_BRANCH_BASE_MINOR=5
+                    RAW_VERSION_LINK=$(echo ${GIT_REPO%.git} | sed -e "s:github.com:raw.githubusercontent.com:g")
+                    wget ${RAW_VERSION_LINK} -O ${WORKSPACE}/VERSION-${BUILD_NUMBER}
```

Comment:
What does RAW_VERSION_LINK resolve to on a sample run?

Reply:
https://ps.cd.percona.com/job/test-percona-server-5.7-pipeline/8/console